### PR TITLE
Check consonant clusters

### PR DESCRIPTION
**Pull request by chrzyki**
Command as proposed and implemented by @LinguList for checking (potentially) problematic consonant clusters.

### CODE REVIEW

**Comment by FredericBlum** [comment]
I have two ideas for potential additions:

- Add a summary per dataset, so we can identify the ˈworstˈ cases that need immediate attention
- Add an argument for the length of clusters to be checked. 3 could be the default, but maybe we want ot check for 4 or 5 first, to identify the most problematic cases?

Also, I am thinking that maybe the Segments would be a more relevant output than the (unstandardized) form. It seems difficult at some points to identify the problematic part in the forms. Maybe we could output both? What do you think?

**Review by LinguList — APPROVED** [review]
I agree with the proposal.

File: src/pylexibank/commands/consonant_clusters.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
+"""
+Check for (potentially) problematic consonant clusters > 2.
+"""
+
+from clldutils.clilib import Table, add_format
+from cldfbench.cli_util import with_dataset, add_catalog_spec
+from pylexibank.cli_util import add_dataset_spec
+import collections
```
Comment (xrotwang) [inline src/pylexibank/commands/consonant_clusters.py:8]: Could we move imports from stdlib to the top, please?

**Review by xrotwang — CHANGES_REQUESTED** [review]

File: src/pylexibank/commands/consonant_clusters.py (hunk 1 above)
Comment (chrzyki) [reply src/pylexibank/commands/consonant_clusters.py]: My bad, fixed :)

**Review by xrotwang — APPROVED** [review]

**Pull request merged**